integrationTests/vm/wasm/queries/queries_test.go

```diff
@@ -0,0 +1,206 @@
+//go:build !race
+
+// TODO remove build condition above to allow -race -short, after Wasm VM fix
```

Comment:
Question for review: is this still applicable?

Reply:
yes, can not be taken out for now :(